src/features/surveys/components/surveyForm/OptionsQuestion.tsx

```diff
@@ -0,0 +1,79 @@
+import { FC } from 'react';
+import {
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Select,
+} from '@mui/material';
+import {
+  ZetkinSurveyOption,
+  ZetkinSurveyOptionsQuestionElement,
+} from 'utils/types/zetkin';
+
+const OptionsQuestion: FC<{ element: ZetkinSurveyOptionsQuestionElement }> = ({
+  element,
+}) => {
+  return (
+    <FormControl>
+      <FormLabel
+        id="demo-radio-buttons-group-label"
```

Comment:
Doesn't this mean that this element will have the same ID for all "option" (multiple-choice) questions, i.e. that the same ID will repeat in the DOM? I'm guessing that can cause problems. Maybe using the `element.id` somewhere in the value of this ID attribute could make it unique?

Reply:
Looks like another side-effect of all the fun high-paced work at Code Camp. Have replaced this with correct values in 76f95f42ca0. I'm grateful that this issue has brought me back to this area of the code as it's brought the form group semantics to my attention and I think there might be some more details to perfect here which I'll revisit once the more pressing stuff is dealt with.